### PR TITLE
First pass at a primer

### LINKED ISSUE #252: Usage Scenarios should not be in the spec.
The [Usage Scenarios](https://github.com/cloudevents/spec/blob/master/spec.md#usage-scenarios) section is valuable context to the project but is too verbose to be part of the spec. The spec should be concise and technical. 

I suggest breaking the Usage Scenarios into another document. I would be happy to do this if the group agrees.

### PR DESCRIPTION
**Pull request by duglin**
For the most part this just moves around existing documentation - but I did add a few bits of text just to help it flow a bit.  As people think of additional education material they want to add we can add it to `primer.md`

Closes #252
Closes #139 
Closes #53

Signed-off-by: Doug Davis <dug@us.ibm.com>

### CODE REVIEW

**Comment by rperelma** [comment]
@duglin I noticed you somehow dropped Adobe: https://github.com/cloudevents/spec/pull/231

**Comment by duglin** [comment]
@rperelma on geez - sorry about that, fixed.

File: primer.md [hunk 1]
```diff
@@ -0,0 +1,572 @@
...
+In order to achieve the stated goals, the working group will attempt to
+constrain the number of metadata attributes they define in CloudEvents. To
+that end, attributes defined by this working group will fall into three
+ categories:
+- required
+- optional
+- extensions
+
+As the category names imply, "required" attributes will be the ones that
+the working group considers vitial to all events in all use cases, while
```
Comment (markfisher) [inline primer.md]: typo: vitial

Comment (duglin) [reply primer.md]: thanks - fixed

**Comment by n3wscott** [comment]
nice, already ahead of the game.

fixes https://github.com/cloudevents/spec/issues/252

**Comment by duglin** [comment]
@cathyhongzhang look under Value Proposition in primer.md

**Comment by duglin** [comment]
Decided to pick-up the text from https://github.com/cloudevents/spec/pull/53 at the same time.

**Comment by duglin** [comment]
Approved on 7/5 call

**Pull request merged**